### DISCUSSION TITLE
Upgrading Git-LFS whilst keeping Git the same version ?

### DISCUSSION THREAD

**Discussion by questiontime-github** [git-lfs/git-lfs#4379] [General]
Hi,
For various reasons I need to keep the Bitbucket and associated Git version i'm running the same version as it is now - however I want to patch for the Git-LFS vulnerability. I couldn't find any information on if I can upgrade my version of Git-LFS to the latest patched version i.e  v2.13.2 whilst keeping my Git version at 2.16.2.windows.1 (current Git-LFS is 2.3.4.0).
The environment is Windows.

Thanks for any help.

**Comment by bk2204** [comment]
Hey,

If you want to update your Git LFS version, I'd reinstall your Git for Windows version without the Git LFS component and then install the Git LFS binary manually into your `PATH`.  Currently we don't offer a way to replace the version offered with Git for Windows.

**Reply by KalleOlaviNiemitalo** [reply]
If the machine is dedicated to Bitbucket Server, then you could just not install Git LFS at all. Before Bitbucket Server 5.9, it did not even work properly if Git LFS was installed; see [BSERV-9616](https://jira.atlassian.com/browse/BSERV-9616) and [Git Large File Storage (Data Center and Server 5.8)](https://confluence.atlassian.com/bitbucketserver058/git-large-file-storage-947168250.html):

> The user running the Bitbucket Server instance should not have the Git LFS client executable installed.

That incompatibility has been fixed but [Supported Platforms (Data Center and Server 7.9)](https://confluence.atlassian.com/bitbucketserver/supported-platforms-776640981.html) still does not say that you'd _need_ Git LFS on the server.
